doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -373,22 +377,42 @@ This chapter describes the system parameters used by the X-Road configuration pr
 | minimum-global-configuration-version | 2                                 | Minimum supported global configuration version on configuration proxy. Change this if old global configuration versions need to be supported. |
 
 ### Signer parameters: `[signer]`
-| **Name**                       | **Default value**                       | **Description**    |
-|--------------------------------|-----------------------------------------|--------------------|
+
+| **Name**                       | **Vanilla value**                       | **Description** |
+|--------------------------------|-----------------------------------------|-----------------|
 | ocsp-response-retrieval-active | false <br/> _(see Description for more information)_ | This property is used as an override to deactivate periodic OCSP-response retrieval for components that don't need that functionality, but still use signer. <br/><br/> Values: <br/> `false` - OCSP-response retrieval jobs are never scheduled <br/> `true` - periodic OCSP-response retrieval is active based on ocspFetchInterval. **Note that if the entire property is missing, it is interpreted as true.** <br/><br/>  This property is delivered as an override and only for the components where the OCSP-response retrieval jobs need to be deactivated. The property is missing for components that require OCSP-response retrieval to be activated. |
 | ocsp-cache-path                | /var/cache/xroad                        | Absolute path to the directory where the cached OCSP responses are stored. |
 | enforce-token-pin-policy       | false                                   | Controls enforcing the token pin policy. When set to true, software token pin is required to be at least 10 ASCII characters from at least tree character classes (lowercase letters, uppercase letters, digits, special characters). (since version 6.7.7) |
 
 [1] See also [*http://www.quartz-scheduler.org/documentation/quartz-1.x/tutorials/crontrigger*](http://www.quartz-scheduler.org/documentation/quartz-1.x/tutorials/crontrigger).
-
```

Comment:
Add extra new line/carriage return here for clarity. So that it is [1] and [2] lines are not written together.